docs/jwst/extract_1d/description.rst

```diff
@@ -239,25 +247,27 @@ direction perpendicular to dispersion.
 
 Extraction for 3D IFU Data
 --------------------------
-For IFU cube data, 1D extraction is contolled by a different set of EXTRACT1D
-reference file parameters.
-Note that for an extended source, anything specified in the reference file
+For IFU cube data, 1D extraction is controlled by a different set of EXTRACT1D
+the RA/DEC target location indicated by the header or the center of the cube if the header
+target location is undefined. For extended source data, anything specified in the reference file
 or step arguments will be ignored; the entire image will be
-extracted, and no background subtraction will be done.
+extracted, and no background subtraction will be done. For IFU data the ``use_source_posn`` option is not used. 
```

Comment:
Really? Why isn't use_source_posn used when we know the data are using a point-source nod pattern? If the hardwired position of the extraction region in the ref file is set to the center of the cube, isn't that going to lead to bad extractions when nodding/dithering is involved and hence the source is not centered spatially in the cube?

Reply:
The center of the extraction region is set by the target ra/dec and if that is not given then it is set by the center of the IFU. If this method does not work for nodded IFU data then we will need to update the ifu.py routine in extract.py. Nothing about use_source_posn is used in the routine.
@drlaw1558 comments ? I think the extract ifu.py is optimized for MIRI (does MIRI MRS even use Nodding ?)

Reply:
OK, what you described is basically the same functionality as use_source_posn. For non-IFU modes, when use_source_posn=True, the center of the extraction is computed from the target RA/Dec, and when use_source_posn=False the position is based on the params from the extract1d ref file. So I guess we're OK. It just might be confusing to users that use_source_posn only applies to non-IFU data.

Reply:
I was confused myself how the extraction parameters for different modes was working. It took me a while to understand it (and I am still learning). One reason why I explicitly said 'use_source_posn was not used in IFU data' was so that the user would not use it. Or try to use it and wonder why it did nothing.  Maybe we need to revisit this with NIRSpec Nodded IFU data. I will leave that to the NIRSpec experts.

Reply:
If, as you said above, the center of the extraction region is set by the target ra/dec, that should work equally well for both MRS and NRS IFU dithers/nods, because it will force the extraction aperture to follow the source around the field of view. Unless of course what you described above is only being done for MRS data and not for NRS IFU, in which case that would be a problem. I would hope/assume that both types of IFU data are treated the same way.